src/posts/pangeo-ml-ecosystem-2023/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'The Pangeo Machine Learning Ecosystem in 2023'
+date: '2024-01-19'
```

Comment:
```suggestion
title: 'The Pangeo Machine Learning Ecosystem'
date: '2024-04-04'
```
Let's try to get this out this week, and maybe we could leave the year off the title to avoid confusion 🚀